From 9f04f9285cad083e23c2d2665787b355a28b6feb Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Tue, 5 Jun 2012 23:11:35 -0400
Subject: [PATCH] configure: Make executable

I had this locally, but forgot to send out the amended patch.

Signed-off-by: Colin Walters <walters@verbum.org>
---
 0 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 configure

diff --git a/configure b/configure
old mode 100644
new mode 100755
-- 
1.7.7.6

